.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": ["bloq", "prettier"],
+  "ignorePatterns": "packages/*/dist/*",
+  "overrides": [
+    {
+      "extends": ["bloq/typescript", "prettier"],
+      "files": ["packages/**/*.ts"]
+    },
+    {
+      "extends": ["bloq/vitest"],
```

Comment:
```suggestion
      "extends": ["bloq/vitest", "prettier"],
```

Reply:
Updated in [b76dfb1](https://github.com/bloq/eth-packages/pull/8/commits/b76dfb1d362e8997046adbbdfcc8c2023103c95b)